app.js

```diff
@@ -1,7 +1,11 @@
+// Javascript core Module
+
 const path = require('path')
 const http = require('http')
 const fs = require('fs');
 
+// JavaScript Third Party Module (Production Environment)
```

Comment:
You have added a comment for Prod env but I can't see any code which might be picking environment from the command line while running the code.
Please check

Reply:
Ok Sir , I will check

Reply:
Sure.. and once done.. mark this conversation as resolved and re request me for review. 😉🙂